x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -76,16 +78,18 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     return await this.baseClient.delete(type, id, options);
   }
 
-  public async find(options: SavedObjectsFindOptions) {
+  public async find<T = unknown>(
+    options: SavedObjectsFindOptions
+  ): Promise<SavedObjectsFindResponse<T>> {
```

Comment:
__nit:__ can we just explicitly specify type for `baseClient.find<T>` below and don't define full response type here?

* Same question for `get` and `bulkGet` below.
* Same question for `get`, `find` and `bulkGet` in Spaces SO wrapper below.

Reply:
done